x-pack/docs/en/rest-api/repository-source-only.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[repository-src-only]]
+=== Source Only Repository
+
+The Source Only repository adds support for creating `_source` only snapshots using any other
+available repository as it's storage backend. This allows using {ref}/modules-snapshots.html[Snapshot/Restore]
+to create incremental, storage optimized, and minimal snapshots of an index.
+
+
+[[repository-src-only-usage]]
+==== Configuration
+
+The `_source` only repository always requires a delegate repository to be used as it's storage backend.
+In order to use the `fs` repository:
+
+[source,js]
+-----------------------------------
+PUT _snapshot/my_src_only_repository
+{
+  "type": "source",
+  "settings": {
+    "delegate_type": "fs",
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// CONSOLE
+
+Since the `_source` only repository doesn't snapshot any index or doc-values structures but only stored
+fields and index metadata, it's required to reindex the data after the restore process. The restored index is read-only
```

Comment:
I think we need to make this way more prominent (warning note?)

Reply:
yeah I will rework the docs a bit .